src/page-layout/SignInLayout/SignInLayout.jsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+import classNames from "classnames/bind";
+import styles from "./SigninLayout.module.css";
+import Link from "next/link";
+import { useForm } from "react-hook-form";
+
+const LOGO_IMAGE = "images/linkbrary.svg";
+export function SignInLayout() {
+  const {
+    register,
+    handleSubmit,
+    formState: { isSubmitting, isSubmitted, errors },
+  } = useForm();
+  const cx = classNames.bind(styles);
+  return (
+    <body className={cx("body")}>
+      <header>
+        <Link className={cx("logo-link")} href="/">
+          <img
+            className={cx("header-logo")}
+            src={LOGO_IMAGE}
+            alt="홈으로 연결된 Linkbrary 로고"
+          />
+        </Link>
+        <p className={cx("header-message")}>
+          회원이 아니신가요?
+          <Link className={cx("header-link")} href="/signup">
+            회원 가입하기
+          </Link>
+        </p>
+      </header>
+      <div className={cx("sign-box")}>
+        <form
+          id="form"
+          className={cx("sign-form")}
+          onSubmit={handleSubmit((data) => alert(JSON.stringify(data)))}
+        >
+          <div className={cx("sign-inputs")}>
+            <div className={cx("sign-input-box")}>
+              <label className={cx("sign-input-label")}>이메일</label>
+              <input
+                id="email"
+                className={cx("sign-input")}
+                placeholder="이메일을 입력해 주세요"
+                {...register("email", {
+                  required: "이메일을 입력해 주세요",
+                  pattern: {
+                    value: /\S+@\S+\.\S+/,
+                    message: "이메일 형식에 맞지 않습니다.",
+                  },
+                })}
+                aria-invalid={
+                  isSubmitted ? (errors.email ? "true" : "false") : undefined
+                }
+              />
+              {errors.email && (
+                <p
+                  role="alert"
+                  id="email-error-message"
+                  className={cx("error-message")}
+                >
+                  {errors.email.message}
+                </p>
+              )}
+            </div>
+            <div className={cx("sign-input-box", "sign-password")}>
+              <label className={cx("sign-input-label")}>비밀번호</label>
```

Comment:
## `label`과 `id`를 연결해보는건 어떨까요?

Reply:
```suggestion
    <label for="password" className={cx("sign-input-label")}>비밀번호</label>
```

라벨은 다음과 같은 이점이 있습니다:
`<label>` 을 [`<input>` (en-US)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) 요소와 연결하면 몇 가지 이점이 있습니다:

- label 텍스트는 텍스트 입력과 시각적으로 관련이 있을뿐만 아니라 프로그래밍적으로도 관련이 있습니다. 예를 들어, 화면리더기(screenreader) 는 폼 입력(form input)에서 label 을 읽어서 보조기술(assistive technology) 사용자가 입력해야하는 텍스트가 무엇인지 더 쉽게 이해할 수 있게 합니다.
- 관련 label 을 클릭해서 input 자체에 초점을 맞추거나 활성화를 시킬 수 있습니다. (활성되어서)늘어난 누를 수 있는 영역(hit area)은 터치스크린 사용자를 포함해 입력하려하는 모든 사람에게 이점을 줍니다.
> 출처: [MDN](https://developer.mozilla.org/ko/docs/Web/HTML/Element/label)